admin-dev/themes/new-theme/js/pages/product/edit/product-partial-updater.js

```diff
@@ -167,10 +186,57 @@ export default class ProductPartialUpdater {
   getFormDataAsObject() {
     const formArray = this.$productForm.serializeArray();
     const serializedForm = {};
+
     formArray.forEach((formField) => {
-      serializedForm[formField.name] = formField.value;
+      let {value} = formField;
```

Comment:
```suggestion
      let value = formField.value;
```

If eslint allows you to do this I think it's easier to read

Reply:
No it wont, and this is a bad practice, always prefer object destructuring :)